doc/source/structures/misc/resource_transfer.rst

```diff
@@ -0,0 +1,100 @@
+.. _resource_transfer:
+
+Resource Transfer 
+=================
+
+Usage is covered elsewhere <../commands/resource_transfer.html>`__
```

Comment:
When you are ready for a release later let me know and I'll take one last pass at parsing the docs across all the recent PRs rather than looking at them one PR at a time.